### PR TITLE
Add GNU Emacs refcard to more/free-programming-cheatsheets.md file un…

### PR DESCRIPTION
**Pull request by AR10X**
…der ide/editors category

## What does this PR do?
Add resource(s)

### Description
Added GNU Emacs refcard/cheatsheet from the official site.

### Why is this valuable (or not)?
Great reference card for Emacs users.

### How do we know it's really free?
Official reference card from GNU.org is available for free to download as PDF.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: more/free-programming-cheatsheets.md [hunk 1]
```diff
@@ -80,6 +80,7 @@
 ### <a name="ide--editores"></a>IDE / Editors
 
 * [Editor VI - Guia de Referência](https://aurelio.net/curso/material/vim-ref.html) - Aurelio Marinho Jargas
+* [GNU Emacs Refcard](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - Official PDF from [GNU.org](https://www.gnu.org/home.en.html)
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:83]: Use this snippet as fix. Ot should solve linter faults :wink:

- PDF links should have `(PDF)` format mark

```markdown
* [GNU Emacs Reference Card](https://www.gnu.org/software/emacs/refcards/pdf/refcard.pdf) - GNU.org (PDF)
```

**Comment by davorpa** [comment]
Resolves #5627

**Comment by ps173** [comment]
I don't see any problem with pr why is it not getting accepted

**Comment by eshellman** [comment]
much needed.
Thanks, and welcome to free-programming-books!

**Review by davorpa — APPROVED** [review]
Resolve #5627

**Comment by davorpa** [comment]
> I don't see any problem with pr why is it not getting accepted

Please, @ps173 don't pressure, I have also other life. Your feedback: I had to go out for a task in the middle of the review

**Comment by ps173** [comment]
@davorpa I apologise for doing so. I tend to act highly unprofessional but I assure you I don't intend to pressure you

**Pull request merged**